tasks/prime.py

```diff
@@ -38,6 +38,35 @@ def fetch_mto_updates(self):
         else:
             logger.info(f"ℹ️ Num MTOs returned: {len(json_body)}")
 
+    @tag("mtoServiceItem", "createMTOServiceItem")
+    @task
+    def create_mto_service_item(self):
+        payload = {
+            "moveTaskOrderID": "5d4b25bb-eb04-4c03-9a81-ee0398cb779e",
+            "mtoShipmentID": "475579d5-aaa4-4755-8c43-c510381ff9b5",
+            "modelType": "MTOServiceItemDDFSIT",
```

Comment:
do we possibly want to test more types of service items? If so, let's address this when [MB-2968](https://dp3.atlassian.net/secure/RapidBoard.jspa?rapidView=25&projectKey=MB&modal=detail&selectedIssue=MB-2968) is complete.

Reply:
that's a good point. For example, domestic crating requires the dimensions of the crate. So what's required in the payload differs, based on the service items.

Reply:
oh yeah, this is a case I've been working on specifically! it's a little tricky to vary the model type, but it's a fun problem for sure